src/app/app.module.ts

```diff
@@ -28,6 +28,9 @@ import { SearchResultsComponent } from './search-playground/search-results/searc
 import { environment } from '../environments/environment';
 import { StillImagePlaygroundComponent } from './still-image/still-image-playground.component';
 import { ViewerPlaygroundComponent } from './viewer-playground/viewer-playground.component';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
```

Comment:
organize imports

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808